live-examples/js-examples/globalprops-decodeuricomponent.html

```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js">function containsEncodedComponents(x) {
+  // ie ?,=,&,/ etc
+	return (decodeURI(x) !== decodeURIComponent(x));
```

Comment:
I think this is tab-indented, which it should not be.

(for indentation it seems you generally use 2 spaces, right? Again this is a place where having a standard and linting would be great. Looking at existing JS code samples on MDN, they seem to use 2. So 2 sounds good to me.)

Reply:
Well spotted :) Yes 2 spaces - I recently turned invisible characters on in my editor (Atom) as copying and pasting from the interactive editor was causing issues. Unfortunately this one slipped through the net. I suspect there might be others out there - but we can remedy with a PR dedicated to formatting.